dev_requirements.txt

```diff
@@ -3,6 +3,8 @@ qiskit-terra~=0.18.2
 qiskit-aer~=0.8.2
 qiskit-ibmq-provider~=0.16.0
 pyquil~=2.28.0
+pennylane-qiskit~=0.18.0
+pennylane>=0.18
```

Comment:
Do you specify anywhere in the docs a minimum version for soft dependencies?

Reply:
@crazy4pi314, @rmlarose, @nathanshammah  what do you think?

In principle it would be nice to add info about minimum versions but there are some downsides:

- It can easily become outdated.
- In general, we actually don't know what are the minimum versions since we only test the specific versions  pinned in `dev_requirements.txt`.